README.md

```diff
@@ -39,37 +45,54 @@ You can contribute by proposing new goals for a focus area, or new questions for
 or new metrics for answering those questions. Or by helping to refine goals, and questions.
 
 
-## Meetings
 
-We meet every Wednesday at 11am CDT (usually 18:00 CET, but beware different switches to Summer time in EU and US, [check your local time](http://www.thetimezoneconverter.com/?t=11am&tz=Chicago&)) in the CHAOSS Zoom room https://unomaha.zoom.us/j/720431288 (Meeting ID for [dial in](https://unomaha.zoom.us/zoomconference?m=DKGo2mmIuOv9xSjphoGZZmYKxr5HFrS9): 720 431 288)
 
-Usually, we try to make decissions mainly during the last meeting of each month,
-so that decisions can be brought when convenient to the monthtly main CHAOSS meeting,
-which is the first meeting of each month. In all the meetings we may have an agenda,
-but random issues can be raised if time allows, after we're done with the agenda.
-Whenever possible, and specially if a decission needs to be made,
-ensure that issues / pull requests about the subject matter were opened some time before the meeting,
-so that anyone had the opportunity of commenting on them, and make up their mind about that subject matter.
+TODO: Fill out a short section that links to D&I metrics repo and explains how this repo fits into overall CHAOSS metrics effort.
```

Comment:
Again, this is about D&I, it should be about the GMD wg.

Reply:
Link to D&I: https://github.com/chaoss/wg-diversity-inclusion

Reply:
Alternative description, to fix the TODO (preliminarty version, to be completed in further iterations):

> This is the repoitory for the CHAOSS GMD (Growth-maturity-decline) Working Group. We are defining GMD metrics for software development projects, and their reference implementations. For that, we are currently focusing in some focus areas, and then following a goal-question-metric approach, defining goals relevant in each of the focus areas,  questions that help to address those goals, and metrics that help to answer those questions.